test/data/config.template.yaml

```diff
@@ -0,0 +1,7 @@
+name: node/install-test
+version: 1.2.0
+maintainer: foo@bar.com
+description: Template to install and test npm packages
+steps:
```

Comment:
added a `config`
not sure what sort of validation we're expecting for a `lifespan`